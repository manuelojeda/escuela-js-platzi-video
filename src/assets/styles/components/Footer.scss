.footer {
    align-items: center;
    background-color: #8f57fd;
    display: flex;
    height: 100px;
    width: 100%;
  }
  
  .footer a {
    color: white;
    cursor: pointer;
    font-size: 14px;
    padding-left: 30px;
    text-decoration: none;
  }
  
  .footer a:hover {
    text-decoration: underline;
  }