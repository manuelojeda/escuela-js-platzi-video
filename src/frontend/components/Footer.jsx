import React from 'react';
import '../assets/styles/components/Footer.scss';

const Footer = () => (
  <footer className='footer'>
    <a href='/'>Terminos de uso</a>
    <a href='/'>Declaraci√≥n de privacidad</a>
    <a href='/'>Centro de ayuda</a>
  </footer>
);

export default Footer;
