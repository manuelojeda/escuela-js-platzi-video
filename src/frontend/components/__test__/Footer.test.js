import React from 'react'
import { create } from 'react-test-renderer'
import { render, shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Footer from '../Footer'

configure({ adapter: new Adapter() });

describe('Footer testing', () => {
  test('Match Snapshot', () => {
    const footer = create(<Footer />)
    expect(footer.toJSON()).toMatchSnapshot();
  });

  test('footer haves class .footer', () => {
    const footer = shallow(<Footer />)
    const footerElem = footer.find('footer')
    expect(footerElem.hasClass('footer')).toBe(true);
  });

  test('footer haves 3 a', () => {
    const footer = render(<Footer />)
    expect(footer.find('a')).toHaveLength(3);
  });

})
