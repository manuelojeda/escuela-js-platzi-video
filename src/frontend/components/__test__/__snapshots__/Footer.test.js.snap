// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing Match Snapshot 1`] = `
<footer
  className="footer"
>
  <a
    href="/"
  >
    Terminos de uso
  </a>
  <a
    href="/"
  >
    Declaraci√≥n de privacidad
  </a>
  <a
    href="/"
  >
    Centro de ayuda
  </a>
</footer>
`;
